src/pages/resources/migration/ios/migrate-to-ios-4x.md

```diff
@@ -0,0 +1,93 @@
+| [Places Service](https://experienceleague.adobe.com/docs/places/using/home.html) | ![Version](https://img.shields.io/github/v/release/adobe/aepsdk-places-ios.svg?label=AEPPlaces&logo=apple&logoColor=white&color=orange&sort=semver&filter=4*) |
+| [Adobe Analytics](../../../solution/adobe-analytics/index.md) | ![Version](https://img.shields.io/github/v/release/adobe/aepsdk-analytics-ios.svg?label=AEPAnalytics&logo=apple&logoColor=white&color=orange&sort=semver&filter=4*) |
+| [Adobe Analytics - Media Analytics for Audio & Video](../../../solution/adobe-media-analytics/index.md) | ![Version](https://img.shields.io/github/v/release/adobe/aepsdk-media-ios.svg?label=AEPMedia&logo=apple&logoColor=white&color=orange&sort=semver&filter=4*) |
+| [Adobe Target](../../../solution/adobe-target/index.md) | ![Version](https://img.shields.io/github/v/release/adobe/aepsdk-target-ios.svg?label=AEPTarget&logo=apple&logoColor=white&color=orange&sort=semver&filter=4*) |
+| [Adobe Campaign Standard](../../../solution/adobe-campaign-standard/index.md) | ![Version](https://img.shields.io/github/v/release/adobe/aepsdk-campaign-ios.svg?label=AEPCampaign&logo=apple&logoColor=white&color=orange&sort=semver&filter=4*) |
+| [Adobe Campaign Classic](../../../solution/adobe-campaign-classic/index.md) | ![Version](https://img.shields.io/github/v/release/adobe/aepsdk-campaignclassic-ios.svg?label=AEPCampaignClassic&logo=apple&logoColor=white&color=orange&sort=semver&filter=4*) |
+| [Adobe Audience Manager](../../../solution/adobe-audience-manager/index.md) | ![Version](https://img.shields.io/github/v/release/adobe/aepsdk-audience-ios.svg?label=AEPAudience&logo=apple&logoColor=white&color=orange&sort=semver&filter=4*) |
+
+## Migrate from Adobe Experience Platform 3.x SDK libraries
+
+If you have implemented Adobe Experience Platform 3.x SDK libraries, then this guide will help you understand the steps required to migrate your implementation to the Experience Platform 4.x SDK libraries. In summary, you'll need to:
+
+1. [Update dependencies](#update-dependencies)
+2. [Update outdated API references](#update-outdated-api-references)
+
+### Update dependencies
+
+#### Manual library import
+
+If you are importing SDK libraries manually, make sure to update your libraries by downloading the most recent 4.x binaries from their respective Github repositories.
+
+#### CocoaPods
+
+If you are using CocoaPods to manage your Adobe Experience Platform Mobile SDK dependencies, the following example shows you how to update to Experience Platform 4.x SDKs in your `Podfile`.
+
+```ruby
+# Update pod dependencies to 4.x version
+# pod 'AEPCore', '~> 3.0'
+# pod 'AEPLifecycle', '~> 3.0'
+# pod 'AEPIdentity', '~> 3.0'
+# pod 'AEPSignal', '~> 3.0'
+# pod 'AEPUserProfile', '~> 3.0'
+
+  pod 'AEPCore', '~> 4.0'
+  pod 'AEPLifecycle', '~> 4.0'
+  pod 'AEPIdentity', '~> 4.0'
+  pod 'AEPSignal', '~> 4.0'
+  pod 'AEPUserProfile', '~> 4.0'
+```
```

Comment:
SUPER minor nit - any reason that we have extra spaces here in front of all the pod?

Reply:
I added an extra space so that all the pod statements line up. 
https://developer.adobe.com/client-sdks/previous-versions/documentation/migrate-to-swift/#cocoapods